unit_test/sycl/Test_SYCL_Batched.cpp

```diff
@@ -0,0 +1,9 @@
+#ifndef TEST_SYCL_BATCHED_CPP
+#define TEST_SYCL_BATCHED_CPP
+#define KOKKOSKERNELS_SYCL_BATCHED_TESTS
```

Comment:
I am suspecting that this variable is not used, I hope I did not forget to remove it from the other backends...

Reply:
It's also in `CUDA` where I copied it from. :slightly_smiling_face: